apps/web/src/modules/report/hooks/useReport/index.ts

```diff
@@ -0,0 +1,74 @@
+import { zodResolver } from '@hookform/resolvers/zod'
+import { GetJobDto } from '@modela/dtos'
+import { useNoti } from 'common/context/NotiContext'
+import { apiClient } from 'common/utils/api'
+import { useRouter } from 'next/router'
+import { FormEventHandler, useCallback, useEffect, useState } from 'react'
+import { SubmitHandler, useForm } from 'react-hook-form'
+
+import { IReportSchemaType, ReportSchema } from './schema'
+
+const useReport = () => {
+  const router = useRouter()
+  const { jid } = router.query
+  const [loading, setLoading] = useState(false)
+  const [jobName, setJobName] = useState('')
+  const { displayNoti } = useNoti()
+  const { control, handleSubmit } = useForm<IReportSchemaType>({
+    criteriaMode: 'all',
+    resolver: zodResolver(ReportSchema),
+    defaultValues: {
+      description: '',
+    },
+  })
+
+  const handleSuccess: SubmitHandler<IReportSchemaType> = useCallback(
+    async ({ ...data }) => {
```

Comment:
Do we have any concern that making us to use spread operator ?

Reply:
idk, I just copy this from your signup form 😂. should I change this?

Reply:
```suggestion
    async (data) => {
```

Reply:
Or maybe 
```suggestion
    async ({ description }) => {
```
then change from `data.description` to `description`